NorthstarDLL/util/version.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
 extern char version[16];
-extern char NSUserAgent[32];
+extern char NSUserAgent[256];
```

Comment:
What's up with this? Doesn't seem particularly relevant to this PR?

Reply:
It's defined at 256 bytes long in the relevant .cpp file, which meant this definition here was wrong. It should have thrown a compile error before, but somehow didn't and nobody noticed. And my PR's changes made it complain that 32 here mismatched 256 in the other place...

Reply:
Makes sense, fair enough